kr/loadingDescription.xml

```diff
@@ -20,215 +20,215 @@
 	<tip id="17" startTime="0-0-0-0-0-0" endTime="0-0-0-0-0-0" text="You can view real estate pricing through Tria's $map:02000123$."/>
 	<tip id="18" startTime="0-0-0-0-0-0" endTime="0-0-0-0-0-0" text="If you withdraw all items in the house, it will be moved to the mover's storage warehouse"/>
 	<tip id="19" startTime="0-0-0-0-0-0" endTime="0-0-0-0-0-0" text="When you are at your home, you can press 'B' to modify your house."/>
-	<tip id="20" startTime="0-0-0-0-0-0" endTime="0-0-0-0-0-0" text="You can place items down in land or inside your building."/>
+	<tip id="20" startTime="0-0-0-0-0-0" endTime="0-0-0-0-0-0" text="You can place items down in land or inside your building.">
```

Comment:
Yeah file was not up to date, missing this commit 79b43951eced504ade2e751f4b9ed21a03645fa2
I'll fix it